Lib/idlelib/calltip_w.py

```diff
@@ -74,16 +82,30 @@ def showtip(self, text, parenleft, parenright):
             int, self.anchor_widget.index(parenleft).split("."))
 
         super().showtip()
+        self.tipwindow.wm_attributes("-topmost", 1)
 
         self._bind_events()
 
     def showcontents(self):
         """Create the call-tip widget."""
-        self.label = Label(self.tipwindow, text=self.text, justify=LEFT,
+        self.label = Label(self.tipwindow, text=self.text, font=self.anchor_widget['font'])
+        self.label.pack()
+        label_w, label_h = widget_size(self.label)  # get the old version of tooltip window size
```

Comment:
```suggestion
        label_w, label_h = widget_size(self.label)
```

I would rename the variables `old_w` and `old_h` instead.